.github/workflows/cd-publish.yml

```diff
@@ -3,38 +3,98 @@ name: CD - Publish for Blazor / FluentUI
 on:
   workflow_dispatch
 
+env:
+  OUTPUT_PATH: ${{ github.workspace }}/.nupkg
+  NUGET_SOURCE: "https://api.nuget.org/v3/index.json"
```

Comment:
Remove dotnet specific versions. we don't have a requirement here.

Reply:
this is the default nuget link

Reply:
But, this is already defined in `nuget.config` and the default, so it should not be needed in this file.  Correct me if I'm wrong.

Reply:
Yes. they are the same. but it will not cause any issue.

Reply:
It may not cause an issue, but it's code duplication.  It's best to try and limit code duplication to improve maintainability.   What file, if any is this required in?  Which is it optional in?

Reply:
Spoke with @Ibrahimmaga and he left it in `./nuget.conf` as the only source of truth removing duplication.